magda-web-client/src/config.js

```diff
@@ -22,7 +22,12 @@ const serverConfig: {
     feedbackApiBaseUrl?: string
 } =
     window.magda_server_config || {};
-
+//this below const enables suggest a Dataset when enabled
+export const enableSuggestDatasetPage = true;
+export const correspondenceApiReportUrl =
+    "https://magda-dev.terria.io/api/v0/correspondence/send/dataset/:datasetId/report";
+export const correspondenceApiUrl =
```

Comment:
follow the format of the next line otherwise this won't adapt for production environment

Reply:
so like...
```javascript
export const correspondenceApiUrl = serverConfig.correspondenceBaseUrl || fallBackApiHost + "correspondence/send/...."
```

Stupid question, but I don't get why we do this? It appears to have no bearing on anything?

Reply:
There's a bit of snakes and ladders but essentially in Production it will expect to access the API on the same domain name using this config: https://github.com/TerriaJS/magda/blob/master/deploy/kubernetes/config/web.json#L3

Reply:
Yep this is super important!!